package.json

```diff
@@ -66,11 +73,6 @@
     "prettier": "^1.16.4",
     "react": "16.9.0-alpha.0",
```

Comment:
The react library is kept here to avoiding the installation of @storybook's custom version (storybook 5.1.0 uses react's  16.8) and therefore has problems running projects.

One possible solution is to move storybook dependencies to all projects under the `devDepencies` entry, not sure if this could be a better solution.

@webmaster128 thoughts?

Reply:
The issue is that 16.9.0-alpha.0 [is not compatible](https://jubianchi.github.io/semver-check/#/^16.8.4/16.9.0-alpha.0) to the storybook's version range `^16.8.4` because it is a pre-release version.

When we use react version "^16.8.6" instead, everything is resolved to a single installation:

```
react@^16.7.0, react@^16.8.4, react@^16.8.6:
  version "16.8.6"
  resolved "https://registry.yarnpkg.com/react/-/react-16.8.6.tgz#ad6c3a9614fd3a4e9ef51117f54d888da01f2bbe"
  integrity sha512-pC0uMkhLaHm11ZSJULfOBqV4tIZkx87ZLvbbQYunNixAAvjnC+snJCg0XQXn9VIsttVsbZP/H/ewzgsd5fxKXw==
  dependencies:
    loose-envify "^1.1.0"
    object-assign "^4.1.1"
    prop-types "^15.6.2"
    scheduler "^0.13.6"
```

So, do we really need this pre-release version of react?

Reply:
yes, otherwise we suffer a nasty error testing hooks, and when using material-ui components.

Reply:
Good, the other option is to force storybook to use 16.9.0-alpha.0 as well:

In root `package.json` add

```
  "resolutions": {
    "react": "16.9.0-alpha.0",
    "react-dom": "16.9.0-alpha.0"
  }
```

Reply:
I do really like this! Will update now.

Just FYI, I have just tested test suite using 16.8 and the problem is React cannot deal with async TestUtils.act calls, which are necessary to perform user interactions (and wait for them).

Reply:
Done!!